src/main/java/seedu/address/logic/commands/meeting/MeetCommand.java

```diff
@@ -60,23 +61,20 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         List<Person> lastShownList = model.getFilteredPersonList();
         Set<Id> attendees = new HashSet<>();
-        String feedback = "";
         for (Index index : toMeet.getIndexes()) {
-            try {
-                attendees.add(lastShownList.get(index.getZeroBased()).getId());
-            } catch (IndexOutOfBoundsException exception) {
-                feedback += " Unknown contact: " + index.getOneBased() + "\n";
-            } finally {
-                continue;
+            if (index.getZeroBased() >= lastShownList.size()) {
+                throw new CommandException(String.format(MESSAGE_INVALID_PERSON_DISPLAYED_INDEX,
+                        index.getOneBased()));
             }
+            attendees.add(lastShownList.get(index.getZeroBased()).getId());
```

Comment:
Will this be able to handle `IndexOutOfBoundsException` thrown by `index.getZeroBased()`?

Reply:
An `IndexOutOfBounds` exception would be thrown during the construction of the Index if its value is less than 0, which should be handled in `ParserUtil#parseIndex()` when the Strings are being parsed to Indexes.